app.js

```diff
@@ -19,6 +22,24 @@ app.use(bodyParser.urlencoded({
   extended: true,
 }));
 
+
+app.use(express.static(path.join(__dirname, "public")));
+app.use(session({
+  secret: 'this is not a secret ;)',
+  resave: false,
+  saveUninitialized: false
+  }));
+app.use(passport.initialize());
+app.use(passport.session());
+
+passport.serializeUser(function(user, done) {
```

Comment:
We probably want to actually serialize our user. Serialize user determines what we actually store in our session database. Right now this stores the entire user object, which is redundant since we already store the whole object in our normal database. Normally serializeUser has done(null, user.id) and then deserializeUser findsUserById and calls done with the found user.

it should looks like:
```
passport.serializeUser(function(user, done) {
    done(null, user.id);
);

passport.deserializeUser(function(userId, done) {
     User.findById(userId, (err, user)=>{done(null, user)})
);
```